src/components/Listbox/Listbox.tsx

```diff
@@ -107,36 +152,116 @@ export function Listbox({
 
         scrollIntoView(focusTarget, scrollableRef.current);
       }
-    }, 15),
+    },
     [],
   );
 
+  const handleScrollIntoViewDebounced = debounce(handleScrollIntoView, 15);
```

Comment:
is 15ms a bit short of a duration here?

Reply:
This is the duration originally set when `Autocomplete` was added so I left it as is. 
What duration would you recommend @kyledurand?

Reply:
All good to leave as is